packages/nimiq-vitepress-theme/src/assets/code-blocks.css

```diff
@@ -290,4 +294,47 @@ pre {
       display: none;
     }
   }
+
+  /* ComponentPreview spacing */
+  .nq-raw[class*='f-my-lg'] {
+    .nq-raw > div {
+      padding-top: 1rem;
+    }
+  }
+
+  /* Alternative targeting for ComponentPreview */
+  [data-radix-tabs-content] {
+    padding-top: 1rem;
+  }
+
+  /* Target ComponentPreview content areas */
+  .nq-raw .grow {
+    padding-top: 1rem;
+  }
+
+  /* Direct targeting for ComponentPreview tabs content */
+  .nq-raw [role='tabpanel'] {
+    padding-top: 1rem;
+  }
+
+  /* Specific targeting for ComponentPreview preview content */
+  .nq-raw [role='tabpanel'][data-state='active'] {
+    padding-top: 1rem;
+  }
+
+  /* Target the preview content area specifically */
+  .nq-raw [data-radix-tabs-content][data-state='active'] {
+    padding-top: 1rem;
+  }
+
+  /* More specific targeting for ComponentPreview content */
+  .nq-raw > div > div {
+    padding-top: 1rem;
+  }
+
+  /* Direct targeting for TabsContent */
+  .nq-raw [data-radix-tabs-content] {
+    padding-top: 1rem !important;
+  }
```

Comment:
Small tip.

You can use Nested CSS. This block of code is the same as the following:

```css

.nq-raw {
	.grow {
    padding-top: 1rem;
  }

  /* Direct targeting for ComponentPreview tabs content */
  [role='tabpanel'] {
    padding-top: 1rem;
  }

  /* Specific targeting for ComponentPreview preview content */
  [role='tabpanel'][data-state='active'] {
    padding-top: 1rem;
  }

  /* Target the preview content area specifically */
  [data-radix-tabs-content][data-state='active'] {
    padding-top: 1rem;
  }

  /* More specific targeting for ComponentPreview content */
  & > div > div {
    padding-top: 1rem;
  }

  /* Direct targeting for TabsContent */
  [data-radix-tabs-content] {
    padding-top: 1rem !important;
  }
}
```